symfony/webpack-encore-bundle/1.9/webpack.config.js

```diff
@@ -23,7 +23,7 @@ Encore
     .addEntry('app', './assets/app.js')
 
     // enables the Symfony UX Stimulus bridge (used in assets/bootstrap.js)
-    .enableStimulusBridge('./assets/controllers.json')
```

Comment:
This `assets/controllers.json` file is managed by Flex, so you cannot rename it here.

Reply:
OK. So what can I do?

Reply:
Well, either not renaming the controllers.json file, or first building a way to make this filename configurable in Flex to allow new projects to use the new name while existing projects would still use `assets/controllers.json` until they migrate.

Reply:
Migrate? I was thinking these recipes here are a *one-time* installation "helper", and this is (more or less) what's called "Flex" ;-) But obviously there's more to it... So where is this Flex-thing ;-) configured?

Reply:
[Flex](https://github.com/symfony/flex) is composer plugin, that installs recipes and manages a few files automatically (`config/bundles.php` and `assets/controllers.json`)

Reply:
Thanks! It's coming from this line: https://github.com/symfony/flex/blob/main/src/PackageJsonSynchronizer.php#L111

So in order to make this change, both repos need to be updated *simultaneously*?

For existing users it shouldn't make a difference, cause the path in `.enableStimulusBridge()` (in `webpack.config.js`) overrides, right?

Reply:
Well, for existing users, they need their Flex plugin to keep managing the `assets/controllers.json` file until the time they rename it. That's why I said it needs a migration strategy for the Flex plugin.